scripts/console

```diff
@@ -19,8 +20,16 @@ then
     if [ "${1:-}" = "--help" ]
     then
         usage
+    elif [ "${1:-}" = "--with-airflow" ]
```

Comment:
Did you try leaving this as it was, but just including the `-f docker-compose.airflow.yml` argument? Worried that adding `--with-airflow` to `console` will always be forgotten.